src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString
+    .split(';')
+    .map(property => property.split(':'))
+    .filter(property => property.length === 2);
+  const callback = (object, property) => {
```

Comment:
Можеш деструктурувати одразу ключ і значення
```suggestion
  const callback = (object, [key, value]) => {
```